package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Just leaving a comment here to discuss package.lock files

https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json

> "This file is intended to be committed into source repositories"

Sounds like we should commit it then. It'll ensure that local development builds and concourse are identical. It shouldn't impact end users.